ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/message/AlterDatabaseHandler.java

```diff
@@ -77,9 +79,22 @@
         alterDbDesc = new AlterDatabaseSetOwnerDesc(actualDbName, new PrincipalDesc(newDb.getOwnerName(),
             newDb.getOwnerType()), context.eventOnlyReplicationSpec());
       }
+      Path metricPath = null;
+      ReplicationMetricCollector metricCollector = null;
+      try{
+        metricPath = ReplUtils.getMetricPath(context, context.hiveConf);
```

Comment:
you are passing the context already. hiveconf is part of that

Reply:
hiveConf has default access in Context, can't be accessed by ReplUtils.